cluster/examples/kubernetes/ceph/pool.yaml

```
@@ -57,7 +57,7 @@ spec:
   # quota in bytes and/or objects, default value is 0 (unlimited)
   # see https://docs.ceph.com/en/latest/rados/operations/pools/#set-pool-quotas
   # quotas:
-    # maxBytes: 10737418240 # 10GiB = 10 * 1024 * 1024 * 1024
+    # maxSize: 10Gi # valid suffixes include Ki, Mi, Gi, Ti, Pi
```

It might not need quotes, but for readability it helps show it's a string type
```suggestion
    # maxSize: "10Gi" # valid suffixes include Ki, Mi, Gi, Ti, Pi
```

---

If no suffix is specified, it would be interpreted as bytes, right?

---

Yes, I have tested that (and made the needed change to the regex), if a fractional number is used by mistake it is rounded up so that shouldn't be a problem